### ISSUE TITLE
Source broken

### ISSUE THREAD

**Issue by cquijano** [kszaq/OpenELEC.tv#35]
Im not able to download que kernel source:

```
--2016-05-16 17:56:50--  http://s805.eu/sources/linux-amlogic-3.10-4e597d3.tar.xz
Resolving s805.eu (s805.eu)... failed: Name or service not known.
wget: unable to resolve host address 's805.eu'
--2016-05-16 17:56:50--  http://sources.openelec.tv/mirror/linux/linux-amlogic-3.10-4e597d3.tar.xz
Resolving sources.openelec.tv (sources.openelec.tv)... 82.220.2.33
Connecting to sources.openelec.tv (sources.openelec.tv)|82.220.2.33|:80... connected.
HTTP request sent, awaiting response... 404 Not Found
2016-05-16 17:56:50 ERROR 404: Not Found.
```

**Comment by kszaq** [comment]
Change the site address to sx05.eu

**Comment by kszaq** [comment]
Fixed in 5095f2f0a68fabdd62fd40c94fa1f221443eaa64

**Comment by cquijano** [comment]
Thanks!
